app/controllers/CharitiesController.scala

```diff
@@ -49,4 +49,12 @@ class CharitiesController @Inject()(servicesConfig: ServicesConfig,
     implicit request =>
       Future.successful(Redirect(controllers.routes.HelloWorldController.helloWorld()))
   }
+
+  def loadDifficulties(): Action[AnyContent] = Action.async {
+    implicit request => {
+      Future.successful(InternalServerError(views.html.error_template(Messages("charities_elig.notEligible"),
+        Messages("charities_elig.notEligible"),
```

Comment:
I would suggest Messages should be converted at the view level rather than controller.